data/pages.yml

```diff
@@ -12,6 +12,8 @@
       url: "index"
     - title: "Your First Ember App"
       url: "first-app"
+    - title: "Glossary"
+      url: "glossary"
     - title: "Core Concepts"
       url: "core-concepts"
```

Comment:
IMO just like a traditional technical book, put the Glossary at the end?

I've found that most of the guides put a sprinkle of what different terms are.
I think the glossary is more of a reference back rather than an introduction?